pyanaconda/modules/payloads/source/repo_files/repo_files.py

```diff
@@ -0,0 +1,66 @@
+#
+# Kickstart module for Repo files payload source.
+#
+# Copyright (C) 2020 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+from pyanaconda.payload.dnf.utils import REPO_DIRS
+
+from pyanaconda.modules.payloads.constants import SourceType
+from pyanaconda.modules.payloads.source.source_base import PayloadSourceBase
+from pyanaconda.modules.payloads.source.repo_files.repo_files_interface import \
+    RepoFilesSourceInterface
+from pyanaconda.modules.payloads.source.repo_files.initialization import \
+    SetUpRepoFilesSourceTask
+
+from pyanaconda.anaconda_loggers import get_module_logger
+log = get_module_logger(__name__)
+
+
+class RepoFilesSourceModule(PayloadSourceBase):
+    """The Repo files source payload module."""
+
+    @property
+    def type(self):
+        """Get type of this source."""
+        return SourceType.REPO_FILES
+
+    def is_ready(self):
+        """This source is ready for the installation to start."""
+        res = True
+        log.debug("Source is set to %s ready state", res)
+        return res
```

Comment:
Please change this to just `return True`.
Also add:
``FIXME: Returning True is correct here but it will prevent payload from setting sources. Find better solution.``

Reply:
Done!